### PR TITLE
Create project layout

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Comment by awgreene** [comment]
Please review @kevinrizza, @galletti94

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+The Operator Courier is used to build, validate and push Operator Artifacts.
+
+## Building the tool locally with pip
+```bash
+$ pip install --user .
+
+$ operator-courier
+Hello, world. The build feature is currently under construction.
+Hello, world. The push feature is currently under construction.
+Hello, world. The validate feature is currently under construction.
```
Comment (kevinrizza) [inline README.md]: This is just going to change as soon as any of this is implemented, right? Why not just leave the output out of the readme?

**Comment by kevinrizza** [comment]
Just one small comment, otherwise lgtm.

File: README.md (hunk 1 above)
Comment (awgreene) [reply README.md]: Addressed in the latest commit.

**Pull request merged**